tests/unit/importer/test_submission.py

```diff
@@ -166,11 +184,12 @@ def _create_spreadsheet_json():
 
 class IngestSubmitterTest(TestCase):
 
-    @patch('ingest.importer.submission.Submission')
+    @patch('ingest.importer.submission.ingest_submitter.Submission')
     def test_submit(self, submission_constructor):
         # given:
         ingest_api = MagicMock('mock_ingest_api')
```

Comment:
This is mocked on many tests. Should this be in a setup fixture?

Reply:
Done! Thanks!